packages/sdks/javascript/src/services/UserService.ts

```diff
@@ -1,8 +1,10 @@
 import { UsersApiService } from '../api/generated/apis/users-api';
 import { UsersTeamInvitationsApiService } from '../api/generated/apis/users-team-invitations-api';
 import { UsersPasswordsApiService } from '../api/generated/apis/users-passwords-api';
+import { UsersEnvironmentsApiService } from '../api/generated/apis/users-environments-api';
 
 export class UserService extends UsersApiService {
     public teamInvitations = new UsersTeamInvitationsApiService();
     public passwords = new UsersPasswordsApiService();
+    public environments = new UsersEnvironmentsApiService();
 }
```

Comment:
I'd suggest keeping properties sorted alphabetically:
```suggestion
    public environments = new UsersEnvironmentsApiService();
    public passwords = new UsersPasswordsApiService();
    public teamInvitations = new UsersTeamInvitationsApiService();
}
```

Reply:
I've rearranged it in alphabetical order: https://github.com/devopness/devopness/pull/770/commits/d09ccd007273be7ca2672d0e537464730b0d7d7c